Files/Local Files/Copy a Single file from Fileshare to SFTP/process.json

```diff
@@ -0,0 +1,67 @@
+﻿{
+	"ProcessTemplates": [
+		{
+			"Name": "Copy a single file from a local fileshare to SFTP server",
```

Comment:
Also, we need to recreate the template with no tags in process because of the Frends bug.